testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -748,6 +748,10 @@
   end
 end
 
+Then(/^I should not see a warning sign$/) do
+  raise 'Errors detected in the Products Catalog' unless page.has_no_xpath?("//*[contains(@class, 'fa fa-li fa-exclamation-triangle text-warning')]", wait: 5)
+end
+
```

Comment:
Cool xpath usage

Why this timeout ? Maybe we could use the default timeout for capybara? It's defined as:
```
Capybara.default_max_wait_time = ENV['CAPYBARA_TIMEOUT'] ? ENV['CAPYBARA_TIMEOUT'].to_i : 10
```

Reply:
Agreed, removed the timeout and will look into applying this to some other similar definedsteps